sosia/processing/caching/utils.py

```diff
@@ -1,11 +1,13 @@
 """Module with utility functions for processing and caching data."""
 
 from copy import deepcopy
+from sqlite3 import Connection
+from typing import Iterable
```

Comment:
I think that's deprecated and should be replaced by `collections.abc`. Please have a look.